### ISSUE TITLE
Deploying Cloud Foundry applications in Code Engine: Getting started - documentation

### ISSUE THREAD

**Issue by arpanroy** [ibm-cloud-docs/codeengine#17]
There are several issues:
1. There is an extra '$' in front of "const http = require('http');
2. The application does not get created rather gives "The permission to perform IBM Container Registry operations is insufficient". There was no registry secret defined in my account but IBM cloud documentation should have mentioned it. 
3. After creating secrets and targeting us.icr.io (or de.icr.io) I get same error message: "Internal server error: error code 'internal', error trace 'codeengine-cli-b70608mlz7'. Contact IBM Cloud Support if this error persists.". I'm using IBM Cloud shell.
4. The application, does not get created - as obvious.

**Comment by arpanroy** [comment]
Oh.. forgot to mention that I supplied the new secret by adding --registry-secret option of  ibmcloud ce app create, which is not mentioned in documentation

**Comment by kersten1** [comment]
Hello
Sorry to hear of your difficulties.

1. agree with you on the extra `$`. However, it isn't copied when you copy the sample block. I'll fix the example so that it doesn't show though.
2. Not sure exactly why you got your error, but I'm assuming it is a problem with authenticating to Container Registry. Are you the account owner or admin? You do not have to create a secret as Code Engine will check to see if you have one first and then create one for you if you do not. You can, of course, create a secret and target it when you create an app but it isn't necessary, as explained in the steps after running the `app create` command.
3. Not sure exactly why you are getting the Internal server error but I do not think it is because of the example. 

My recommendation is to create a new namespace in the account in Container Registry. Then, after you target your `sample` project again, delete any artifacts of the app in your project `ibmcloud ce app delete -n myapp` and try running the example again.

**Comment by arpanroy** [comment]
Hi Kersten,

Thanks for the tip, I sensed it is a permission related issue in the container registry and I can't get around it merely by creating a registry secret. No I'm using a paid account for these poc where I have been added recently and I'm not the owner or administrator. While defining the build strategy I observed it is expecting the container repository internal secret. I shall try to create own namespace in Tellus account and try this out again. 
Regards
Arpan
